blocked-edges/4.19.0-ec.1-VsphereStorageMountIssues.yaml

```diff
@@ -0,0 +1,12 @@
+to: 4.19.0-ec.1
+from: 4[.]18[.].*
+url: https://issues.redhat.com/browse/STOR-2486
+name: VsphereStorageMountIssues
+message: Vsphere customers using vSAN file volumes can't mount vSphere shared volumes and nfs volumes which server do not set NFS4ERR_ATTRNOTSUPP
+matchingRules:
+  - type: PromQL
+    promql:
+      promql: |
+        group(cluster_infrastructure_provider{type=~"VSphere|None"})
```

Comment:
Why does this have `None` also btw?

Reply:
It should only apply to vSphere platform. If someone is bringing their own CSI cloudprovider and CSI driver, they may still be affected but IMO blocking metal etc seems wrong.